speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/ConfirmationMenu.cpp

```diff
@@ -0,0 +1,81 @@
+#include <tgfclient.h>
+
+#include "legacymenu.h"
+#include "ConfigEnums.h"
+#include "mediator.h"
+#include "ConfirmationMenu.h"
+#include "mainmenu.h"
+
+static void* MenuHandle = nullptr;
+
+/// @brief tells the mediator to not save experiment data and close SpeedDreams
+static void 
+OnAcceptExit(void* /* dummy */)
```

Comment:
function declaration should be on a single line
```suggestion
static void OnAcceptExit(void* /* dummy */)
```